quickstart/101-azure-api-management-create/providers.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_version = ">=0.12"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~>3.0"
```

Comment:
Will this example really work with any version for the provider 3.0? If not, it should be updated to a more current version.

Reply:
Fixed